components/NavigationDrawer/tests/unit/MDCNavigationDrawerScrollViewTests.m

```diff
@@ -463,4 +463,19 @@ - (void)testBottomDrawerCornersAPIFullScreen {
   XCTAssertEqual(self.drawerViewController.maskLayer.minimumCornerRadius, 3.f);
 }
 
+- (void)testBottomDrawerDynamicSizing {
+  // Given
+  self.fakeBottomDrawer.contentViewController =
+      [[MDCNavigationDrawerFakeTableViewController alloc] init];
+  [self.fakeBottomDrawer viewDidLoad];
+  [self.fakeBottomDrawer cacheLayoutCalculations];
+  CGFloat previousContentHeaderTopInset = self.fakeBottomDrawer.contentHeaderTopInset;
+
+  // When
+  self.fakeBottomDrawer.contentViewController.preferredContentSize = CGSizeMake(200, 200);
+
+  // Then
+  XCTAssertNotEqual(previousContentHeaderTopInset, self.fakeBottomDrawer.contentHeaderTopInset);
```

Comment:
To narrow down this test a bit, can we verify that it's greater than or less than the previous value?

Reply:
done